### DISCUSSION TITLE
Versioning datasets and releases

### DISCUSSION THREAD

**Discussion by erikjohnson24** [aplbrain/BENCHMARK-Metadata#6] [General]
A key missing aspect from our current proposal is versioning of releases. It was suggested in the working group that checkpoint versions with change logs may be well suited to a use case of versioned public releases of datasets for secondary analysis/reproducibility. 

A couple comments that came up: 
Versioning and assessing impact of proofreading on circuit/graph generation and quality
Versioning and tracking of changes are needed in the case changes need to be reversed
Versioning can support tracking of needed updates/changes/corrections from released (static/flattened/agglomerated) versions
Serve as checkpoint and continuous versioning

If there are strong thoughts about versioning, or concrete examples we should leverage for this effort, please chime in!

**Comment by DocSavage** [comment]
I do have strong thoughts about versioning as you might imagine, particcularly when it pertains to data for long-lived curated data that may go through multiple publication rounds.  I guess the question is how to proceed. I'm also seeing some needs due to AI training such that it's important to preserve what's looked at by humans and their level of expertise.
